unittests/optionaltype_regular/testregularapi.h

```diff
@@ -37,6 +37,7 @@ class TestRegularApi : public QObject
 private slots:
     void test();
     void testPolymorphic();
+    void testSerialize();
```

Comment:
Can you do the same in optionaltype_boost_optional and optionaltype_pointer? Just to make sure they all work.

Thanks.

Reply:
Done, although I don't have boost so I didn't test that part